qa/workunits/rados/test_rados_tool.sh

```diff
@@ -181,6 +181,7 @@ for i in `seq 1 5`; do
     run_expect_succ "$RADOS_TOOL" -p "$POOL" setomapval $objname key.$j "$rand_str"
   done
   run_expect_succ --tee "$fname.omap.vals" "$RADOS_TOOL" -p "$POOL" listomapvals $objname
+  run_expect_succ "$RADOS_TOOL" -p "$POOL" clearomap $objname
```

Comment:
this fails the copy pool test, see http://pulpito.ceph.com/kchai-2017-12-02_05:16:05-rados-wip-kefu-testing-2017-12-01-1256-distro-basic-smithi/.

Reply:
thanks for the result, looks like there is a copy pool a few lines later, I'll move the test to test_omap() which has more exclusive omap tests